test/models/operation_test.js

```diff
@@ -122,4 +122,16 @@ describe('Operation', function() {
       assertMixinSpecificationExtensionsInheritance(Operation);
     });
   });
+
+  describe('#security()', function() {
+    it('should return an array of security requirements objects', function() {
+      const d = new Operation(js);
+      expect(Array.isArray(d.security())).to.equal(true);
+      expect(d.security().length > 0).to.equal(true);
```

Comment:
Nitpick: I think you can simplify this with the proper Jest function:
```suggestion
      expect(d.security()).toHaveLength(1);
```

Reply:
`.toHaveLength(1);` was throwing an error, I have update it to `.to.have.lengthOf(1)`